server/src/server.js

```diff
@@ -140,6 +140,7 @@ class Server {
 
     this.add_http_handler('auth_methods', (req, res) => {
       res.writeHead(200, { 'Content-Type': 'application/json' });
+      res.writeHead(200, { 'Content-Type': 'application/json', 'Access-Control-Allow-Origin': opts.access_control_allow_origin });
```

Comment:
Why set `Content-Type` twice? Did you forget to take out the previous `writeHead`?